spec/string_spec.rb

```diff
@@ -3,12 +3,30 @@
 RSpec.describe String do
 
   describe "#numeric?" do
-    it 'determines if a certain string containes only numeric values' do
-      expect('123.123'.numeric?).to be true
-      expect('7654'.numeric?).to be true
-      expect('abc'.numeric?).to be false
-      expect('1a1.2'.numeric?).to be false
-      expect(''.numeric?).to be false
+    context 'when valid float number is given' do
+      it 'should return true' do
```

Comment:
You're almost there. :medal_sports: 

There's no need to wrap each spec in its own context. Using the context description as the spec description will bring it closer to what a good test should look like, provided best practices like http://www.betterspecs.org/#should are applied.